uint8 OPEN_DOOR = 0
uint8 CLOSE_DOOR = 1

Header header
uint16  door_id
uint8   action